pkg/operator/operator.go

```diff
@@ -88,6 +105,16 @@ func (o *Operator) Run() error {
 		<-time.After(initRetryDelay)
 	}
 
+	// Run volume provisioner
+	// The controller needs to know what the server version is because out-of-tree
```

Comment:
I guess we can remove it.. But i think this check is useful because it fails right away when the user tries to create the operator. If we dont have it, the users will see the failure later in the workflow. Perhaps we should bump it to 1.6